src/renderer/widgets/RemovePureProxyModal/model/warning-model.ts

```diff
@@ -80,14 +80,19 @@ sample({
   target: formSubmitted,
 });
 
+sample({
+  clock: formInitiated,
+  target: $warningForm.validate,
```

Comment:
If I get this right, when I open the form it will validate and highlight all the fields right away?
It's required by UI mockups or it's something else?

Reply:
Without this trigger boolean values mark as valid if it not touched. What your suggestions for it? It not highlight fields, but needed for submit button